tests/fuzzer.c

```diff
@@ -3988,6 +4114,11 @@ int main(int argc, const char** argv)
 
     if (nbTests < testNb) nbTests = testNb;
 
+    result = threadPoolTests();
```

Comment:
This invocation of `threadPoolTests()` could be done inside `basicUnitTests()`.
It would make it similar to other unit tests, which can be skipped on demand.

Reply:
Sure, I've added it to the end of `basicUnitTests()`.